### PR TITLE
18622 - Correction - Blank Special Resolution

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:* [/bcgov/entity18622](https://github.com/bcgov/entity/issues/18622)

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-edit-dev--pr-550-g7fny7ws.web.app

File: src/components/SpecialResolution/Resolution.vue [hunk 1]
```diff
@@ -11,16 +11,60 @@
...
+          >
+            <v-icon small>
+              mdi-undo
+            </v-icon>
+            <span>Undo</span>
+          </v-btn>
+        </v-col>
+      </v-row>
+
+      <InstructionalText v-if="(requireCorrectionCheck && isCoopCorrectionFiling) || isSpecialResolutionFiling" />
```
Comment (seeker25) [inline src/components/SpecialResolution/Resolution.vue:62]: move to computed?

File: src/components/SpecialResolution/ResolutionEditor.vue [hunk 2]
```diff
@@ -114,7 +115,7 @@
           </div>
           <div
             v-else
-            v-sanitize.basic="getSpecialResolution.resolution"
+            v-sanitize.basic="resolution"
```
Comment (seeker25) [inline src/components/SpecialResolution/ResolutionEditor.vue:118]: good bug fix

File: src/components/SpecialResolution/Resolution.vue (hunk 1 above)
Comment (Jxio) [reply src/components/SpecialResolution/Resolution.vue]: done

**Review by seeker25 — APPROVED** [review]
LGTM nice work Jia

**Pull request merged**